src/include/kernel/ext_manifest.h

```diff
@@ -95,7 +95,7 @@ struct ext_man_dbg_abi {
 struct ext_man_config_data {
 	struct ext_man_elem_header hdr;
 
-	struct config_elem elems[EXT_MAN_CONFIG_LAST_ELEM];
+	struct config_elem elems[];
```

Comment:
Btw, do we have an equivalent PR for the kernel ?

Reply:
@ktrzcinx and what happens if we send < ```EXT_MAN_CONFIG_LAST_ELEM``` to the kernel, as I'm guessing the kernel is expecting ```EXT_MAN_CONFIG_LAST_ELEM``` or greater ? @kv2019i any comments ?

Reply:
@lgirdwood Kernel is reading the structure with a count, so it will not over read 
https://github.com/thesofproject/linux/blob/967c1f2a2d0bf6d719cfcf0e917b6bebd4474b49/sound/soc/sof/loader.c#L211-L218

Also the kernel header do not have the `struct config_elem elems[EXT_MAN_CONFIG_LAST_ELEM];` it seems always to be `struct config_elem elems[];`

So no kernel patch is needed, this PR make alignment with Kernel now